folium/plugins/draw.py

```diff
@@ -10,28 +13,36 @@
 class Draw(MacroElement):
     """
     Vector drawing and editing plugin for Leaflet.
-
     Parameters
     ----------
     export : bool, default False
         Add a small button that exports the drawn shapes as a geojson file.
-
+    filename : string, default data.geojson
+        Name of geojson file
+    position : string, default 'topleft'
+        Position of control. It can be one of 'topleft', 'toprigth', 'bottomleft', 'bottomright'
+    draw : dict, default None
+        The options used to configure the draw toolbar
```

Comment:
1. Can you link to a page where these options are listed? Same for `edit`.
2. I'm not sure about the names `draw` and `edit`, they are too general. The class is named `Draw`, so an object is probably named `draw`. For these arguments, how about `draw_options` and `edit_options`?